RELEASE_NOTES.md

```diff
@@ -3,6 +3,7 @@
 ## New in git-machete 3.7.2
 
 - fixed: package version retrieval outside of git repository
+- fixed: `git merge-base` works for branches that have no common commit
```

Comment:
I'd put it like this:
```suggestion
- fixed: checking whether a branch is merged to parent works for branches that have no common commit
```

Since generally implementation details (like underlying git commands) are better off not included in release notes